modules/thunder_liveblog/thunder_liveblog.install

```diff
@@ -148,3 +150,28 @@ function thunder_liveblog_install() {
   }
 
 }
+
+/**
+ * Adds image to the list of available paragraphs.
+ */
+function thunder_liveblog_update_8101() {
+  /** @var \Drupal\thunder_updater\Updater $thunderUpdater */
+  $thunderUpdater = \Drupal::service('thunder_updater');
+
+  /** @var \Drupal\field\Entity\FieldConfig $field */
+  $field = FieldConfig::load('liveblog_post.liveblog_post.field_embed_media');
+
+  if ($field) {
```

Comment:
If we let it bubble -> it could block all hooks after this one. So I would rather catch it and log it, check point will stay unchecked and user have to check what is wrong manually anyway. If you find any 81xx update hook that doesn't handle errors properly, please create Trello card for it and tag it accordingly.